src/main/java/com/learn/java/collections/impl/StackImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.learn.java.collections.impl;
+
+import com.learn.java.collections.Stack;
+import sun.awt.image.ImageWatched;
+
+public class StackImpl implements Stack {
+    private int size;
+    private LinkedNode firstNode;
+
+    public StackImpl(){
+        size = 0;
+        firstNode = null;
+    }
+
+    @Override
+    public void push(int element) {
+        LinkedNode newNode = new LinkedNode(element);
+        if(size == 0){
+            firstNode = newNode;
+        }else{
+            newNode.next = firstNode;
+            firstNode = newNode;
+        }
```

Comment:
I would preffer this: ``firstNode = new LinkedNode(element, firstNode)``

Reply:
Thanks for your feedback!